.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+- '0.11'
+before_install:
+- npm install -g istanbul
```

Comment:
Just call istanbul like:

'./node_modules/.bin/istanbul'

then you don't need a global install, just the one from the package's dependencies